after/plugin/compe_treesitter.vim

```diff
@@ -0,0 +1,9 @@
+if exists('g:loaded_compe_treesitter')
+  finish
+endif
+let g:loaded_compe_treesitter = v:true
+
+if exists('g:loaded_compe')
```

Comment:
we should check `nvim-treesitter` is installed or not.
(You can check it in here or compe_treesitter/init.lua)